core/tests/Drupal/Nightwatch/globals.js

```diff
@@ -1,7 +1,16 @@
 const chromedriver = require('chromedriver');
+const settings = require('../../../nightwatch.settings.json');
 
 module.exports = {
   before: (done) => {
+    // Setting up 
+    const baseUrl = settings.BASE_URL || process.env.SIMPLETEST_BASE_URL || process.env.BASE_URL;
```

Comment:
Do we need to check to see that these environment variables are possibly empty?

Reply:
I'm curious, does || prevent against that?